packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -152,6 +173,35 @@ export class CdkToolkit {
       }
     }
   }
+
+  public async destroy(options: DestroyOptions) {
+    const stacks = await this.appStacks.selectStacks(options.stackNames, {
+      extend: options.exclusively ? ExtendedStackSelection.None : ExtendedStackSelection.Downstream,
+      defaultBehavior: DefaultSelection.OnlySingle
+    });
+
+    // The stacks will have been ordered for deployment, so reverse them for deletion.
+    stacks.reverse();
+
+    if (!options.force) {
+      // tslint:disable-next-line:max-line-length
+      const confirmed = await promptly.confirm(`Are you sure you want to delete: ${colors.blue(stacks.map(s => s.name).join(', '))} (y/n)?`);
+      if (!confirmed) {
+        return;
+      }
+    }
+
+    for (const stack of stacks) {
+      success('%s: destroying...', colors.blue(stack.name));
+      try {
+        await destroyStack({ stack, sdk: options.sdk, deployName: stack.name, roleArn: options.roleArn });
+        success('\n ✅  %s: destroyed', colors.blue(stack.name));
```

Comment:
I'm part afraid of the kind of messaging this will be on the whole experience... Should make sure all the messaging when you `cdk deploy` is "deployesque".